deployments/vpu_plugin/overlays/xlink/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 bases:
 - ../../base/
-patches:
+patchesStrategicMerge:
```

Comment:
I left this one in as the file is already using `patchesJson6902` (which is also being moved into the `patches` field) to prevent confusion